src/widgets/hits-per-page-selector/__tests__/hits-per-page-selector-test.js

```diff
@@ -56,8 +56,9 @@ describe('hitsPerPageSelector()', () => {
     };
   });
 
-  it("doesn't configure anything", () => {
-    expect(widget.getConfiguration).toEqual(undefined);
+  it('does configure the default hits per page if specified', () => {
+    expect(typeof widget.getConfiguration).toEqual('function');
+    expect(widget.getConfiguration()).toEqual({});
```

Comment:
Maybe we should test the actual configuration if there is a default value? Also we should test that the widget throws an error if there is more than one default value.